mlrun/model_monitoring/model_monitoring_batch.py

```diff
@@ -25,6 +25,7 @@
 import pandas as pd
 import v3io
 import v3io.dataplane
+import v3io_frames
```

Comment:
hopefully this is removed in the upcoming versions right? (there is no v3io in CE :) )

Reply:
yes, all of these v3io imports will be applied only if we the user is not in the CE